LanguageFeatures/Patterns/constant_A03_t02.dart

```diff
@@ -23,87 +23,80 @@
 /// literals explicitly marked const. Likewise with set and map literals versus
 /// map patterns.
 ///
-/// @description Check list, map and set literals with type argument specified
-/// in a constant patterns
+/// @description Check list, map and set literals in a constant patterns. Test
+/// if-case statement
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=patterns
 
 import "../../Utils/expect.dart";
 
 String testList(List value) {
-  switch (value) {
-    case const <int>[1, 2]:
-      return "[1, 2]";
-    case const <String>["3", "4"]:
-      return "['3', '4']";
-    case const <double>[]:
-      return "[]";
-    default:
-      return "default";
+  if (value case const [1, 2]) {
```

Comment:
The type argument is tricky here, cf. https://github.com/dart-lang/language/issues/2637.

Reply:
Similarly for several other locations below with the same structure.